### DISCUSSION TITLE
End of life for packages

### DISCUSSION THREAD

**Discussion by mukund42** [apertureless/vue-chartjs#1086] [Q&A]
Hello Community,
We are in the process of applying for FDA certification and need to include information about the end-of-life dates for packages. Could someone please inform me if there are any available details regarding the end of support for vue-chartjs version 3.5.x?

**Comment by apertureless** [comment]
Hey @mukund42 . `vue-chartjs` 3.5 is the version for vue 2 and chart.js < 3.0.0 which is 2 major chart.js versions behind.

You can consider it EoL already. There will be no support for it.
